packages/react/package.json

```diff
@@ -61,15 +61,15 @@
     "docgen": "pnpm typedoc && docgen --config ./react.docgen.json"
   },
   "dependencies": {
-    "@nhost/hasura-auth-js": "workspace:*",
-    "@nhost/hasura-storage-js": "workspace:*",
     "@nhost/nhost-js": "workspace:*",
     "@xstate/react": "^3.0.1",
     "immer": "^9.0.15",
     "jwt-decode": "^3.1.2",
     "xstate": "^4.33.5"
   },
   "devDependencies": {
+    "@nhost/hasura-auth-js": "workspace:*",
+    "@nhost/hasura-storage-js": "workspace:*",
```

Comment:
Similar question as above, why do we need these packages as dev dependencies? 

We already have `@nhost/nhost-js` as a dependency which includes these dev dependencies.